go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/AbsaOSS/k8s_crd
+
+go 1.14
```

Comment:
- It is new project; start  with GO 1.15 (DockerFile, mod files)

Reply:
done